base/inc/fields/multiple-media.class.php

```diff
@@ -45,6 +57,17 @@ protected function render_field( $attachments, $instance ) {
 			return;
 		}
 
+		// Ensure thumbnail_dimensions are valid. 
+		if (
+			empty( $this->thumbnail_dimensions ) ||
+			empty( $this->thumbnail_dimensions[0] ) ||
+			empty( $this->thumbnail_dimensions[1] ) ||
+			! is_numeric( $this->thumbnail_dimensions[0] ) ||
+			! is_numeric( $this->thumbnail_dimensions[1] )
+		) {
+			$this->thumbnail_dimensions = array( 64, 64 );
```

Comment:
Maybe having this `array( 64, 64 )` as a class constant or static class variable would be better? It's used in multiple places.